translations/it-IT.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   I: 'io',
+  contexts: {
+    Feature: 'Caratteristica',
+    Scenario: 'lo_scenario',
+    Before: 'Prima',
+    After: 'Dopo',
+    BeforeSuite: 'Prima_della_suite',
+    AfterSuite: 'Dopo_la_suite',
```

Comment:
may we follow `CamelCase` of js 😄

Reply:
Other translation followed like this in these files, So I thought we are following this pattern :)

Reply:
ah okay then, cause `CamelCase` would make us look like js ^^